lib/std/sort.zig

```diff
@@ -502,6 +502,257 @@ test "binarySearch" {
     );
 }
 
+/// Returns the index of the first element in `items` not less than `key`,
+/// or `items.len` if no such element exists.
```

Comment:
This is an incorrect description of what this function does. It returns 0 if the key is less than all items, and items.len if the key is greater than all items.